src/ngx_http_vhost_traffic_status_dump.c

```diff
@@ -115,7 +115,7 @@ ngx_http_vhost_traffic_status_dump_node_write(ngx_event_t *ev, ngx_file_t *file,
     if (node != ctx->rbtree->sentinel) {
         vtsn = (ngx_http_vhost_traffic_status_node_t *) &node->color;
 
-        (void) ngx_write_fd(file->fd, vtsn, sizeof(ngx_http_vhost_traffic_status_node_t));
+        (void) ngx_write_fd(file->fd, vtsn, sizeof(u_char));
```

Comment:
This prevents the vhost_traffic_status_dump function from working normally.

Reply:
@vozlt Indeed, I only fixed the ci settings.